ramls/expired-holds-report.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "The hold expiration report",
+  "description": "The hold expiration report that can be used by staff to clear expired and canceled holds from the shelf and put them back into circulation",
+  "type": "object",
+  "properties": {
+    "requests": {
+      "description": "Expired hold request",
```

Comment:
Why does this not use the standard request structure?

Reply:
The standard presentation of request is suitable for it but we already have a contract with the front-end to generate a report that includes the `expired-holds-request.json` in the report.
The `expired-holds-request.json`  includes all the necessary fields that can be extended in the future.

Reply:
This confuses me. 

> The standard presentation of request is suitable for it but we already have a contract with the front-end to generate a report that includes the expired-holds-request.json in the report.

How can there be a contract established when the work hasn't been completed yet?

> The expired-holds-request.json includes all the necessary fields that can be extended in the future.

Is it at least a compatible subset of a request?

Reply:
> Is it at least a compatible subset of a request?

The structure is different. You can take a look at an example: `expired-holds-report.json`

```
{
  "requests": [
    {
      "requesterName": "Koss, Jazlyn",
      "requesterBarcode": "623317241177726",
      "itemTitle": "14 cows for America",
      "itemBarcode": "5860825104574",
      "callNumber": "65487",
      "requestStatus": "Closed - Pickup expired",
      "holdShelfExpirationDate": "2019-06-12T12:26:16.670+0000"
    }
  ],
  "totalRecords": 1
}
```

Reply:
That's a pity. I'll admit to being really confused by this design decision :-/

Reply:
Changed the response to the standard request representation.  Now I will try to fetch the `callNumber` of the item for the request.

Reply:
updated impl